spec/parliament/request/base_request_spec.rb

```diff
@@ -61,10 +61,10 @@
     end
 
     context 'it accepts query parameters' do
```

Comment:
I have changed this test to check the request made contains the parameters expected in the url.